docs/sources/reference/commandline/cli.md

```diff
@@ -1447,6 +1448,10 @@ Import to docker via pipe and `STDIN`.
 
     $ sudo tar -c . | docker import - exampleimagedir
 
+**Import from a local directory with local file:**
+
+    $ sudo docker import /local/path/to/exampleimage.tgz exampleimagedir
```

Comment:
y, I also don't grok - in the context of `docker import [OPTIONS] URL|- [REPOSITORY[:TAG]]`